### ISSUE TITLE
Open source and complete codebase?

### ISSUE THREAD

**Issue by Console** [craigerl/digipi#6]
Just wondering where the php scripts and web interface code can be found in this repository?

Things like setup.php are referred to in other issues here but aren't present within your repository.

Not looking for the image or compiled code, I can happily get that from patreon, but isn't the entire project meant to be open source?

**Comment by craigerl** [comment]
that's a great question, i didn't upload /var/www/html  -- i'll work on it, thanks

**Comment by martysdomain** [comment]
Visited for the same purpose as @Console. I don't mind paying for the image, but GPL code should be available if the project is intended to be open. Thanks.

**Comment by craigerl** [comment]
Yup, absolutely right, i just pushed /var/www/html which includes the php/menu/web stuff.
